### ISSUE TITLE
support additional_backgrounds properties in static theme wizard

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#7019]
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/theme

Additional backgrounds allow extra images to be placed over the main header image.  To be fully functional the wizard should allow adding the extra images and positioning them.

**Comment by Dragneel7** [comment]
@eviljeff I am in interested, can I work on this issue?

**Comment by eviljeff** [comment]
It's not a good first bug as the solution isn't defined and it needs to be done in the next few weeks.  It also needs you to understand SVGs and static themes in addition to Javascript and jinja2 templates.  We won't turn down a patch but there are many other better issues to work on.

**Comment by muffinresearch** [comment]
See also https://bugzilla.mozilla.org/show_bug.cgi?id=1435191  re the issue filed for setting a limit to the additional backgrounds.

**Comment by AlexandraMoga** [comment]
As discussed with @eviljeff on IRC, the testing focus for this issue was concentrated on regressions around the static theme wizard and on submission and installation of packed static themes that have additional backgrounds.

Results:
1. As far as the wizard is concerned, there were no significant regressions observed
2. I haven't encountered issues during the submission process
3. Installation also works as expected

@eviljeff:
I've tried to create a theme that uses multiple backgrounds and used the alignment properties specified [here](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/theme#properties).
The resulted theme is the following: https://addons-dev.allizom.org/en-US/firefox/addon/more-bckg-edited-nine/
After installation, the browser will display only the images aligned at the top. I'm not sure if I've built my theme the wrong way, or if there is an issue here.
